apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -0,0 +1,165 @@
+import { useEffect, useState } from 'react';
+import { Box, Typography, Link } from '@mui/material';
+import { Public as PublicIcon } from '@mui/icons-material';
+import {
+  Twc2SupportDetail,
+  FacebookLinksSection,
+  getTwc2Detail,
+} from '../services';
+import { Container } from '../common/components';
+import { PrimaryButton } from '../common/components/RoundedButton';
+import './SupportDetailWip.css';
+
+function Header() {
+  return <nav className="support-header">Navigation</nav>;
+}
+
+function BlurbIntro(props: { text: string }) {
+  const { text } = props;
+  return (
+    <section className="support-description-blurb-section">{text}</section>
+  );
+}
+
+function BlurbFacebookLinks(props: FacebookLinksSection) {
+  const { links, title, imageSrc } = props;
+  return (
+    <section className="support-description-blurb-section">
+      <Typography variant="h6">{title}</Typography>
+      {links.map(({ href, text }) => (
+        <div className="support-description-blurb-facebook-link">
+          <img
+            src={imageSrc}
+            className="support-description-blurb-facebook-link-icon"
+          />
+          <Link
+            href={href}
```

Comment:
Yeah I thought so, but I didn't see anything in local dev or CI. I guess it won't matter in this case, since we'll never reorder the elements, but I'll give a dummy key just in case.